stable/datadog/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: datadog
-version: 1.0.5
```

Comment:
Why not 1.1.0?

Reply:
Yeah, I should've done a minor bump to begin with, the most recent version bump I did was indeed a minor one to 1.2.0 now.

Reply:
Ohhhh, I thought when I merged in the remote changes the version had already been bumped to 1.1.1, switching to 1.1.0...